src/blant-utils.c

```diff
@@ -175,10 +180,14 @@ void LoadMagicTable(void)
 
 // You provide a permutation array, we fill it with the permutation extracted from the compressed Permutation mapping.
 // There is the inverse transformation, called "EncodePerm", in createBinData.c.
-void ExtractPerm(unsigned char perm[_k], int i)
-{
+void ExtractPerm(unsigned char perm[_k], Gint_type Gint)
+{   
+    if(_k > 8) {
```

Comment:
It *might* be worth it even to do it for k>=8 to save RAM... maybe...

Reply:
Well yes,
But I'd like to run some time calculations before deciding what should we select